src/shared/sentry/sentryadapter.cpp

```diff
@@ -149,11 +150,10 @@ void SentryAdapter::transportEnvelope(sentry_envelope_t* envelope,
   sentry_free(sentry_buf);
 
   auto t = new TaskSentry(qt_owned_buffer);
-  TaskScheduler::scheduleTask(t);
+  TaskScheduler::scheduleTaskNow(t);
```

Comment:
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/308692c7ee9d3bda41d8d6e4cd9af9dadc83c659/src/shared/taskscheduler.h#L20-L22
Footgun check: Is surety at 100%?